cbr-service-profile/main.tf

```diff
@@ -66,10 +66,6 @@ module "cbr_zone" {
 locals {
   rule_contexts = [{
     attributes = [
-      {
```

Comment:
Vincent by removing this are we not opening it to any type of endpoint.  Is that the intention here?